.github/maintainers_guide.md

```diff
@@ -108,33 +108,47 @@ If you want to test the package locally you can.
 
 #### test.pypi.org deployment
 
-[TestPyPI](https://test.pypi.org/) is a separate instance of the
-Python Package Index that allows you to try distribution tools and
-processes without affecting the real index. This is useful with changes that
-relate to the package itself, example the contents of the `pyproject.toml`
+[TestPyPI](https://test.pypi.org/) is a separate instance of the Python Package
+Index that allows you to try distribution tools and processes without affecting
+the real index. This is useful with changes that relate to the package itself,
+example the contents of the `pyproject.toml`
 
 The following can be used to deploy this project on <https://test.pypi.org/>.
 
 ```zsh
-# Set the new version with SLACK_CLI_HOOKS_VERSION
-SLACK_CLI_HOOKS_VERSION="1.2.3" ./scripts/deploy_to_test_pypi.sh
+
+./scripts/deploy_to_test_pypi.sh
 ```
 
 #### Development Deployment
 
-Releases for this library are automatically generated off of git releases. Before
-creating a new release, ensure that everything on a stable branch has landed, then
-[run the tests](#run-all-the-unit-tests).
-
-1. Create a new GitHub Release from the
+Releases for this library are automatically generated off of git releases.
+Before creating a new release, ensure that everything on a stable branch has
+landed, then [run the tests](#run-all-the-unit-tests).
+
+1. Create a branch in which the development release will live:
+   - Bump the version number in adherence to
+     [Semantic Versioning](http://semver.org/) and
+     [Developmental Release](https://peps.python.org/pep-0440/#developmental-releases)
+     in `slack_bolt/version.py`
+     - Example the current version is `1.2.3` a proper development bump would be
+       `1.2.3.dev0`
+     - `.dev` will indicate to pip that this is a
+       [Development Release](https://peps.python.org/pep-0440/#developmental-releases)
+     - Note that the `dev` version can be bumped in development releases:
+       `1.2.3.dev0` -> `1.2.3.dev1`
+   - Commit with a message including the new version number. For example
+     `1.2.3.dev0` & Push the commit to a branch where the development release
+     will live (create it if it does not exist)
+     - `git checkout -b future-release`
+     - `git commit -m 'version 1.2.3.dev0'`
+     - `git push future-release`
```

Comment:
Always fascinating to see the ways to push upstream haha! No change needed, I'm just a big fan of

```
$ git push -u origin future-release
```